elasticsearch/docker_util.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python3
```

Comment:
Since this script is not run directly from the shell, the shebang line can be removed.
(Doesn't hurt anything.)

Reply:
Done